cantools/subparsers/generate_c_source.py

```diff
@@ -84,4 +85,7 @@ def add_subparser(subparsers):
     generate_c_source_parser.add_argument(
         'infile',
         help='Input database file.')
+    generate_c_source_parser.add_argument(
+        '--sender_name',
```

Comment:
I suggest that we rename this to `--node` as we want to generate sender and receiver code for given node. `--sender_name` is too specific to sending. Receiving is just as important.

Reply:
I actually had that at the beginning. Happy to change.